demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -762,7 +762,7 @@ def handle_incorrect_reference_value(self, task_id, values, task_name, section_n
         Returns: True if the references are correct
         """
         is_valid = True
-        split_values = values.split(',')
+        split_values = values.split(',') if isinstance(values, str) else []
```

Comment:
is it either a list of comma separated values, or nothing? is there no option for a single value?

Reply:
@dorschw if its of single value (i.e. no ","), the split func will return a list with single value